connection.go

```diff
@@ -161,6 +161,11 @@ func (c *Connection) Init() error {
 
 	// Tell the kernel not to use pitifully small 4 KiB writes.
 	initOp.Flags |= fusekernel.InitBigWrites
+
+	if c.cfg.EnableAsyncReads {
```

Comment:
When was this introduced in Linux?

What are the requirements on the client-side, if any, for users?

Can this be enabled by default for everyone or are there downsides?

Reply:
https://github.com/jacobsa/fuse/commit/348ed9e731d013c43ea857ac6a9e1330e99faaa0

This feature was there in libfuse for quite sometime. But has been disabled for all only in jacobsa/fuse due to some errors that occurred in gcsfuse. But since jacobsa/fuse is an independent project. It is better if this option is configurable by the user.

Reply:
https://github.com/jacobsa/fuse/commit/348ed9e731d013c43ea857ac6a9e1330e99faaa0
From this link you can see the commit where this was disabled by default for everybody

Reply:
As Far as I know there isn't any downsides. But I think since this commit https://github.com/jacobsa/fuse/commit/d82237972ed9e7b2180ad27b3558f4c746a1fe16 this package seems to fail on rhel 7.5 onwards. But that is totally unrelated to this PR. Just a finding from our use case

Reply:
> But I think since this commit [d822379](https://github.com/jacobsa/fuse/commit/d82237972ed9e7b2180ad27b3558f4c746a1fe16) this package seems to fail on rhel 7.5 onwards. But that is totally unrelated to this PR. Just a finding from our use case

Can you file an issue for this with details please?